docs/get-started/frameworks/nextjs.mdx

```diff
@@ -23,6 +34,21 @@ Storybook for Next.js is a [framework](../../contribute/framework.mdx) that make
 
 ### In a project without Storybook
 
+When you run `storybook init` in your Next.js project, Storybook will automatically detect your project and select the [`@storybook/nextjs-vite`](./nextjs-vite.mdx) framework (which we recommend) unless your project has custom Webpack or Babel configurations that may be incompatible with Vite.
+
+
+<details>
+<summary>If your project has custom Webpack/Babel configurations</summary>
+
+If `storybook init` detects a custom webpack configuration in your nextjs config file or `.babelrc` file, it will prompt you to choose between:
```

Comment:
```suggestion
If `storybook init` detects a custom Webpack configuration in your Next.js config file or `.babelrc` file, it will prompt you to choose between:
```